Assets/Scripts/Modules/Combat/ICombatModule.cs

```diff
@@ -1,8 +1,19 @@
 ﻿/*
  * Authors: Ian Macpherson, Isaiah Mann
- * Description: An interface to handle the turn logic
+ * Description: An interface to handle the combat logic
  */
 
 public interface ICombatModule {
 
+	bool IsTargetInRange (IUnit unit);
```

Comment:
Solid point it used to be IsMonsterInRange then I realized that it could be for anything but forgot to add that.